Update useResetMsgsStates.tsx

Paul-Hammer opened the pull request.

BohdanKushnerov committed:
Merge pull request #29 from BohdanKushnerov/transition-components

add common transition component

Paul-Hammer committed:
Update useResetMsgsStates.tsx

BohdanKushnerov commented:
I've already added most of your recommendations to the project: deleted short names, added a common function 'onReset' for messages, removed all types and interfaces to the parent file except domain models, and added a permission check before starting recording audio messages. Thank you for your help!

Paul-Hammer commented:
Well done! Keep up the good work!

пн, 1 июл. 2024 г. в 12:51, Bohdan Kushnerov ***@***.***>:

> I've already added most of your recommendations to the project: deleted
> short names, added a common function 'onReset' for messages, removed all
> types and interfaces to the parent file except domain models, and added a
> permission check before starting recording audio messages. Thank you for
> your help!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BohdanKushnerov/react-web-messenger/pull/30#issuecomment-2199713571>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APWFSOMFURAMK3EEQI73W6TZKERB7AVCNFSM6AAAAABKCAHCF2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCOJZG4YTGNJXGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
